components/dashboard/src/settings/Account.tsx

```diff
@@ -45,7 +45,7 @@ export default function Account() {
 
         <SettingsPage title='Account' subtitle='Manage account and git configuration.'>
```

Comment:
**issue**: We're still missing the whitespace on the right of this page after a recent change. Ses https://github.com/gitpod-io/gitpod/pull/3582#pullrequestreview-622099641